### ISSUE TITLE
Network Schema out of date

### ISSUE THREAD

**Issue by jabbera** [microsoft/vscode-azurearmtools#28]
Why is the network schema out of date? Creating a resource like:

```
{
      "apiVersion": "2017-06-01",
      "type": "Microsoft.Network/virtualNetworkGateways",
      <SNIP>
}
```

Yields the warning: "Value is not accepted. Valid values: "Microsoft.Network/networkInterfaces"."

**Comment by miensol** [comment]
It's only a guess but it seems that [the schema](https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json) includes a:
```
{"$ref":"https://schema.management.azure.com/schemas/2015-08-01/Microsoft.Network.json#/resourceDefinitions/networkInterfaces"}
```

I'm not sure why only  a subset of resource types is supported 😢

**Comment by StephenWeatherford** [comment]
The extension doesn't have anything to do with these - they're validated by VS Code directly from the schema.  I've reported this issue, we'll see what they say.  Thanks.

**Comment by yevster** [comment]
See a similar error for `Microsoft.Network/virtualNetworks/subnets`

**Comment by StephenWeatherford** [comment]
> Network part is fixed by #653 and https://github.com/Azure/azure-resource-manager-schemas/pull/661/files

**Comment by StephenWeatherford** [comment]
This has been fixed
